docs/products/postgresql/howto/pg-long-running-queries.rst

```diff
@@ -0,0 +1,87 @@
+Detect and terminate long-running queries
+=========================================
+
+Aiven does not terminate any customer queries even if they run indefinitely, but long-running queries can cause issues by locking resources and therefore preventing database maintenance tasks.
+
+To identify and terminate such long-running queries, you can do it from either:
+
+* `Aiven Console <http://console.aiven.io>`_
+* `PostgreSQL® shell <https://www.postgresql.org/docs/current/app-psql.html>`_ (``psql``)
+
+
+Terminate long running queries from the Aiven console
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In the `Aiven Console <http://console.aiven.io/>`_, you can go to the **Current Queries** tab for your service.
+
+.. image:: /images/products/postgresql/pg-long-running-queries.png
+    :alt: PostgreSQL® service overview tab in Aiven's console
+
+
+On the Current Queries page, you can check the query duration and click **Terminate** to stop any long-running queries.
+
+
+Detect and terminate long running queries with ``psql``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can :doc:`login to your service <./connect-psql>`  by running on the terminal ``psql <service_uri>``.  Once connected, you can call the following function on the ``psql`` shell to terminate a query manually::
+
+    SELECT pg_terminate_backend(pid);
+
+
+You can learn more about the ``pg_terminate_backend()`` function from the `official documentation <https://pgpedia.info/p/pg_terminate_backend.html>`_.
```

Comment:
This link is weirdly failing in the check, could you please double check it?

Reply:
Changing the link to be this one instead since this is the official documentation and not the former one
`https://www.postgresql.org/docs/current/functions-admin.html`